### ISSUE TITLE
Last Week in Pony - March 3, 2019

### ISSUE THREAD

**Issue by Theodus** [ponylang/ponylang-website#393]

**Comment by mfelsche** [comment]
The tutorial now has built-in full-text-search.

If anyone with more experience in CSS and JS than me ( @mfelsche ) wants to lend a hand in improving the codebase, we would be enormously grateful. Think of the current setup as jquery and good intentions. How does one even set up a frontend project properly, nowadays? You know? Stuff like this.

**Comment by aturley** [comment]
The recording of the Pony sync for February 26, 2019 is available here:

https://pony.groups.io/g/dev/files/Pony%20Sync/2019_02_26/pony_sync_february_26_2019.m4a

**Comment by SeanTAllen** [comment]
[Version 0.2.2](https://github.com/SeanTAllen/flycheck-pony/releases) of flycheck-pony has been released and is available via melpa and melpa-stable.

**Comment by renatoathaydes** [comment]
I wrote a little bit about Pony in an article I wrote.
Lots of discussion about Pony as usual on Hacker News and Reddit:

https://news.ycombinator.com/item?id=19241427

https://www.reddit.com/r/programming/comments/aucqq8/fearless_concurrency_how_clojure_rust_pony_erlang/

**Comment by SeanTAllen** [comment]
Carl Quinn also known as @cquinn has been invited to become a Pony committer. Everyone welcome Carl to the team!

**Comment by SeanTAllen** [comment]
Pony 0.27.0 has been [released](https://www.ponylang.io/blog/2019/03/0.27.0-released/).

**Comment by SeanTAllen** [comment]
Today, we are announcing that we are "deprecating" our IRC channel. Since we opened up a Slack community, IRC usage has plummeted. Overall community engagement has shot up since we started the Slack. So, the core team recently made the decision to "shut down" the IRC channel. It will continue to exist to prevent squatting but we ask everyone to visit the Slack community instead. 

Get an invite to join here: [https://www.ponylang.io/get-slack-invite](https://www.ponylang.io/get-slack-invite)
